packages/core/src/create-lighthouse-benchmark.ts

```diff
@@ -357,8 +358,10 @@ class LighthouseSampler implements BenchmarkSampler<NavigationSample> {
     _isTrial: boolean,
     _raceCancellation: RaceCancellation
   ): Promise<NavigationSample> {
+    const defaultDesktopSettings = (await eval("import('lighthouse')")).desktopSettings.settings;
```

Comment:
The default desktop configs definede [here](https://github.com/GoogleChrome/lighthouse/blob/main/core/config/desktop-config.js#L11). It sets the right User-agent header. We should extend it instead of overwriting it.

Reply:
Desktop configs can't be extended by adding `extends: 'lighthouse:desktop'`, we must import it and merge it with the configs object.